WordPress/Classes/ViewRelated/Menus/MenuDetailsViewController.m

```diff
@@ -201,6 +203,10 @@ - (void)textFieldValueChanged:(UITextField *)textField
         textField.placeholder = self.menu.name;
     }
     [self tellDelegateMenuNameChanged];
+    if (!self.hasTrackedEditingChanges) {
+        [WPAppAnalytics track:WPAnalyticsStatMenusUpdatedMenuName withBlog:self.menu.blog];
```

Comment:
Seems like bumping the stat here could lead to some false positives. For instance, if the user restores the original value, the change was already made. I guess it depends on if we're wanting to track an actual change or just that text field was used.